command_uninstall.go

```diff
@@ -37,6 +37,18 @@ func cmdUninstall(c *cli.Context) error {
 }
 
 func uninstallPackage(cmd string) error {
+	Outer:
+	for _, packages := range getCommands() {
+		for _, command := range packages.Commands {
+			for _, alias := range command.Aliases {
+				if alias == cmd {
+					fmt.Fprintln(akamai.App.Writer, fmt.Sprintf("Provided name \"%s\" appears to be an alias for installed package \"%s\".", alias, command.Name))
```

Comment:
I don't think we need to output this unless we're verifying it's the correct package. It is possible that two packages provide the same alias, so it might be worthwhile adding this output and confirming if more than one is found, but not otherwise — this does mean you must complete all these loops.

Just to clarify here, while two package commands can have the same alias, the first one we find wins. We should probably abstract out this "find command by alias" logic to it's own method that we can re-use here.